apps/charge-accounts-service/src/operators/interfaces/operator.interface.ts

```diff
@@ -0,0 +1,9 @@
+import { Document } from 'mongoose'
+
+export interface Operator extends Document {
+  readonly firstName: string;
+  readonly lastName: string;
+  readonly emailAddress: string;
+  readonly externallyOwnedAccountAddress: string;
+  readonly smartContractAccountAddress: string;
```

Comment:
that's the owner, right?

Reply:
let's change this to `ownerAccountAddress`
@NikolayRn please validate the terminology here is aligned with rest of the codebase